README.md

```diff
@@ -126,24 +135,32 @@ cd clayui.com
 yarn
 ```
 
-3. Run in development environment
+3. Run in development environment:
 
 ```
 yarn develop
 ```
 
-Run clayui.com on Local LAN
+> You can run clayui.com on Local LAN:
 
 ```
 yarn develop -H 0.0.0.0
 ```
 
-If you want to test in a production environment so that you do not take risks of inconsistencies, in the root folder:
+> If you want to test in a production environment so that you do not take risks of inconsistencies, in the root folder:
```

Comment:
> If you want to test in a production environment so that you do not take risks of inconsistencies, issue the following command in the root directory:

I prefer using "directory" over "folder", because that's usually how they are defined in operating systems and documentation